sycl/include/CL/sycl/accessor.hpp

```diff
@@ -381,39 +391,49 @@ class image_accessor
         T, detail::type_list<cl_int4, cl_float4>>::type::value;
   };
 
-  //  Available only when: (accessTarget == access::target::image ||
-  //  accessTarget == access::target::host_image) && accessMode ==
-  //  access::mode::read
+  // Available only when:
+  // (accessTarget==access::target::image && accessMode==access::mode::read) ||
+  // (accessTarget==access::target::host_image &&
+  //  accessMode==access::mode::read_write)
```

Comment:
```suggestion
  //  (accessMode == access::mode::read || accessMode == access::mode::read_write))
```

Reply:
done

Reply:
NOTE: `accessMode == access::mode::read` check was not added as suggested.

Reply:
added the comment and uploaded it again. 
Sorry about that. I changed the logic in the code, missed to add it in the comment.